### PR TITLE
Add a recipe for snakebite-py3

### PR DESCRIPTION
**Pull request by xylar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge/help-python, this is ready to review.

Just a note that I tried to add a check for the `snakebite` script but it seems to raise an error even with `snakebite --help` or `snakebite --version`, so I'm letting that be for now.

File: recipes/snakebite-py3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - protobuf >2.4.1
```
Comment (mariusvniekerk) [inline recipes/snakebite-py3/meta.yaml:22]: is this a > instead of >= ?

Comment (xylar) [reply recipes/snakebite-py3/meta.yaml:22]: I agree, it's odd, but that's what upstream has:
https://github.com/internetarchive/snakebite-py3/blob/master/setup.py#L45

Comment (mariusvniekerk) [reply recipes/snakebite-py3/meta.yaml:22]: /shrug oh well

**Pull request merged**